CHANGES.rst

```diff
@@ -425,6 +425,10 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Converting an empty table to an array using ``as_array`` method now returns
+  an empty array instead of ``None``. This also means that ``Table() == None``
```

Comment:
Nitpicky, but I think the second part is not needed - it is how you found the bug, but not something most people would do directly (I think).

Reply:
> ... not something most people would do directly (I think).

Agree.